README.md

```diff
@@ -129,6 +129,30 @@ You can redact headers that contain sensitive information by calling `redactHead
 interceptor.redactHeader("Auth-Token", "User-Session");
 ```
 
+### Skip-Inspection 🙅‍♀️🕵️
```

Comment:
nit: We have one emoji on each header. Maybe we should keep that uniform style?

Reply:
Up to you :). I don't mind either way.